src/app/public/modules/numeric/numeric-symbol.ts

```diff
@@ -1,3 +1,6 @@
+/**
+ * @internal
```

Comment:
This is an exported type; probably needs to be in the docs.

Reply:
Should it be exported? Its only used in private methods or once in a self-contained method. I searched ADO, and no SPA is consuming it. Maybe this is an accidental export?

Reply:
Yeah, could be. Disregard.